README.md

```diff
@@ -35,48 +88,228 @@ Here are examples of current capabilities:
+  }
+]
 ```
 
 #### listResourceCategories
 ##### Parameters: **none**
 ##### Request: https://api.spigotmc.org/simple/0.2/index.php?action=listResourceCategories
-##### Response:
+##### Response (truncated):
 ```json
-{
-  "coming": "soon"
-}
+[
+  {
+    "id":"2",
+    "title":"Bungee - Spigot",
+    "description":"Spigot plugins that interact with BungeeCord."
+  },
+  {
+    "id":"3",
+    "title":"Bungee - Proxy",
+    "description":"Bungee plugins that interact directly with the proxy plugins folder."
+  },
+  {
+    "id":"4",
+    "title":"Spigot",
+    "description":"Plugins which work on a standard Spigot install."
+  },
+  {
+    "id":"5",
+    "title":"Transportation",
+    "description":""
+  },
+  {
+    "id":"6",
+    "title":"Chat",
+    "description":""
+  },
+  {
+    "id":"7",
+    "title":"Tools and Utilities",
+    "description":""
+  },
+  {
+    "id":"8",
+    "title":"Misc",
+    "description":""
+  },
+  {
+    "id":"9",
+    "title":"Libraries \/ APIs",
+    "description":""
+  },
+  {
+    "id":"10",
+    "title":"Transportation",
+    "description":""
+  }
+]
 ```
 
 #### getResourceUpdate
 ##### Parameters: `id`, required, the resource update id
-##### Request: https://api.spigotmc.org/simple/0.2/index.php?action=getResourceUpdate&id=2
+##### Request: https://api.spigotmc.org/simple/0.2/index.php?action=getResourceUpdate&id=352711
 ##### Response:
 ```json
 {
-  "coming": "soon"
+  "id":"352711",
+  "resource_id":"6245",
+  "title":"2.10.9 back to normal",
+  "message":"[B][SIZE=6]2.10.9\n[\/SIZE][\/B]\n[LIST]\n[*][SIZE=4]Fixed issues with maven repo regarding Jetbrains annotations when using other IDE's[\/SIZE]\n[\/LIST]\n[SIZE=4]Like this update to pay respects[\/SIZE]\n[LIST]\n[*][SIZE=4]Updated deprecation methods in PlaceholderAPI class. Now normal methods of setPlaceholders will not nag you regardless if you use Player or OfflinePlayer.. This also resolves issues with plugins providing placeholders as both methods pre 2.10.7 are functional from PlaceholderHook. Use onRequest or onPlaceholderRequest.... Doesn't matter until we hit 3.0.0.  [\/SIZE]\n[*][SIZE=4]Fixed a few bugs that probably wont be noticed but if you really care about them you can follow the trail if you are on the hunt [\/SIZE]\n[\/LIST]\n[URL]https:\/\/github.com\/PlaceholderAPI\/PlaceholderAPI\/commits\/master[\/URL]\n\nThis update was mainly focused on people who actually hook into PlaceholderAPI so I hope this resolves any issues you may have with deprecated methods or stuff not working. I really don't want people using the dev repo as a way to release updates however they are a good way for everyone to test. Based on the feedback I felt this was the thing to do as we don't want to break things until PAPI3 drops.\n[SIZE=4]\n\n\n[\/SIZE]"
 }
 ```
 
 #### getResourceUpdates
```

Comment:
Can you please add that there is a page parameter here (same as listResources)? (Same as last comment)